Test/TestFramework/Shared/StubTelemetryInitializer.cs

```diff
@@ -19,7 +20,7 @@ public StubTelemetryInitializer()
         /// <summary>
         /// Gets or sets the callback invoked by the <see cref="Initialize"/> method.
         /// </summary>
-        public TelemetryAction OnInitialize { get; set; }
+        public Action<ITelemetry> OnInitialize = item => { };
```

Comment:
this thing defines the `Action` with the default value of an action that does nothing